tsconfig.json

```diff
@@ -10,12 +11,14 @@
         "strictNullChecks": true,
         "noImplicitReturns": true,
         "noUnusedLocals": true,
-        "noUnusedParameters": true
+        "noUnusedParameters": true,
+        "allowSyntheticDefaultImports": true,
     },
     "exclude": [
         "node_modules"
     ],
     "include": [
-        "src"
+        "src",
+        "types"
```

Comment:
does it really need the d.ts file of itself?

Reply:
Of itself? The d.ts file is for lighthouse logger

Reply:
lol, I've been looking too long at lighthouse-logger I guess :)